src/client/models/tankMovement.js

```diff
@@ -1,7 +1,12 @@
-import constans from './constans';
-var originalPoints = constans.originalPoints;
+import externalVariables from './externalVariables';
+let originalPoints = externalVariables.originalPoints,
```

Comment:
[generateDamage.js](https://github.com/tymochko/pocket-tanks/blob/prepullrequest/src/client/models/generateDamage.js) was checked by you earlier I guess